.devcontainer.json

```diff
@@ -1,14 +1,25 @@
 {
-    "build": { "dockerfile": "Dockerfile" },
-  
-    "customizations": {
-      "vscode": {
-        "extensions": [
-            "quarto.quarto",
-            "ms-python.python"
-          ]
-      }
-    },
-  
-    "forwardPorts": [8888]
+  "image": "mcr.microsoft.com/devcontainers/python:3",
+  "features": {
+    "ghcr.io/rocker-org/devcontainer-features/quarto-cli:1": {
+      "version": "prerelease"
+    }
+  },
+  "forwardPorts": [
+    8888
+  ],
+  "portsAttributes": {
+    "8888": {
+      "label": "Jupyter"
+    }
+  },
+  "vscode": {
```

Comment:
The top level `vscode` field is the old syntax (I think this is not included in the Devcontainer spec).